index.html

```diff
@@ -5,26 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome-books</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/style.css" />
   </head>
   <body>
-    <h1>All Awesome Books</h1>
+    <header>
+      <div class="header-div">
+        <h4>Awesome Books</h4>
+        <nav>
+          <ul class="header-list">
+            <li class="item"><a href="#books">List</a></li>
+            <li class="item"><a href="#add-book">Add new</a></li>
+            <li class="item2"><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+      <h4 class="date">March 22nd 2023, 1:30:00 am</h4>
+    </header>
+    <section id="books">
+      <h1>All Awesome Books</h1>
+      <div class="books-list" id="books-list">
+        <ul class="books" id="books">
+          <!-- Contant dynamically generated with JavaScript -->
+        </ul>
+      </div>
+    </section>
 
-    <div class="books-list" id="books-list">
-      <ul class="books" id="books">
-        <!-- This contant belongs to Java-script -->
+    <section id="add-book">
+      <h2 class="addbook">Add a new book</h2>
+      <div class="book-form">
+        <form>
+          <input type="text" placeholder="title" id="title" required /> <br />
+          <br />
+          <input type="text" placeholder="author" id="author" required /> <br />
+          <br />
+          <button type="submit" id="add-btn">Add</button>
+        </form>
+      </div>
+    </section>
+    <section class="contact" id="contact">
+      <h2>Contact information</h2>
+      <p>
+        Do you have any questions or you just want to say "Hello"?
+      </p>
+      <p class="text">You can reach out to us!</p>
+      <ul class="contact-info">
+        <li>Our email: mail@mail.com</li>
+        <li>Our phone number: 0043586534422</li>
+        <li>Our adress: Streetname 22, 84503 City,Country</li>
       </ul>
```

Comment:
- [ ] It will be great to follow the design given in the wire flame. In your contact page implementation, some of the information are positioned at the left of the page. However, according to the wire flame given [here](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png) all the content is centered. Kindly improve this